public/content/en/basics/classes.md

```diff
@@ -29,13 +28,15 @@ overriding of functions.
     class Bar: Foo {
         override functionFromFoo() {}
     }
+
+In D classes can only inherit from one class.
```

Comment:
"In D,"? (comma) At least I read "In D classes" first and then resumed after I noticed this wasn't intendeded :-)